packages/node_exporter/build.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python3
+
+from pathlib import Path
+from subprocess import run
+import requests
+import tarfile
+
+VERSION = '1.8.2'
```

Comment:
Do we have versions in 2 places?
Jenkins and here.
@c-po is it ok? What will do Jenkins?
Will it clone from the git repo/do nothing and then will download tar.gz anyway?

Reply:
i add the version number (and maybe also the architecture) as an argument to the python script to prevent have it in 2 places?

Reply:
I would assume the later version wins.

https://github.com/vyos/vyos-build/blob/current/vars/cloneAndBuild.groovy#L33-L35

so URL and Commit would not be necessary in the Jenkinsfile at all. @rebortg you can drop the version/URL in the Jenkinsfile, as the build script selects a discrete version which is fine.

Reply:
done